misk/src/main/kotlin/misk/web/interceptors/MetricsInterceptor.kt

```diff
@@ -36,12 +43,22 @@ internal class MetricsInterceptor internal constructor(
     m: Metrics,
     private val caller: @JvmSuppressWildcards ActionScoped<MiskCaller?>
   ) : NetworkInterceptor.Factory {
-    internal val requestDuration = m.histogram(
+    val requestDuration = m.summary(
```

Comment:
why remove `internal`?

```suggestion
    internal val requestDuration = m.summary(
```